File: appjava/src/main/java/com/byoutline/kickmaterial/model/ProjectCreator.java
package com.byoutline.kickmaterial.model;

import org.parceler.Parcel;

/**
 * @author Pawel Karczewski <pawel.karczewski at byoutline.com> on 2015-03-30
 */
@Parcel(Parcel.Serialization.FIELD)
public class ProjectCreator {

    public int id;
    public String name;
    public ProjectCreatorAvatar avatar;
    public CreatorUrls urls;
}

File: app/src/main/java/com/byoutline/kickmaterial/model/ProjectCreator.kt
package com.byoutline.kickmaterial.model

import paperparcel.PaperParcel
import paperparcel.PaperParcelable

/**
 * @author Pawel Karczewski <pawel.karczewski at byoutline.com> on 2015-03-30
 */
@PaperParcel
class ProjectCreator : PaperParcelable {

    var id: Int = 0
    var name: String? = null
    var avatar: ProjectCreatorAvatar? = null
    var urls: CreatorUrls? = null

    companion object {
        @JvmField val CREATOR = PaperParcelProjectCreator.CREATOR
    }
}
